docs/coordinator-migration-guide.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
```

Comment:
I feel that it's enough to have an entire document. We don't need to separate it into modules, such as coordinator, server, client.

Reply:
It's ok for me, updated.

Reply:
The file name is still `coordinator-migration-guide.md`.